misk/src/test/kotlin/misk/eventrouter/EventRouterTest.kt

```diff
@@ -0,0 +1,83 @@
+package misk.eventrouter
+
+import com.google.inject.util.Modules
+import misk.moshi.MoshiModule
+import misk.testing.MiskTest
+import misk.testing.MiskTestModule
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import javax.inject.Inject
+
+@MiskTest(startService = false)
+internal class EventRouterTest {
```

Comment:
we need torture tests where the cluster layout changes, and where we make different routers get out of sync with one another